### PR TITLE
feat(examples): Elasticsearch query using CDK8s+ and CRD

### PR DESCRIPTION
**Pull request by iliapolo**
This also upgrades `jsii-srcmak` to version `0.1.10`.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: examples/typescript/cdk8s-plus-elasticsearch-query/data.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{"index":{}}
```
Comment (eladb) [inline examples/typescript/cdk8s-plus-elasticsearch-query/data.txt:1]: call this `sample-data.txt`

File: examples/typescript/cdk8s-plus-elasticsearch-query/main.test.ts [hunk 2]
```diff
@@ -0,0 +1,224 @@
+import { MyChart } from "./main";
+import { Testing } from "cdk8s";
+
+describe("Placeholder", () => {
+  test("Empty", () => {
+    const app = Testing.app();
+    const chart = new MyChart(app, "test-chart");
+    const results = Testing.synth(chart);
+    expect(results).toMatchInlineSnapshot(`
```
Comment (eladb) [inline examples/typescript/cdk8s-plus-elasticsearch-query/main.test.ts:9]: Prefer external snapshots for big things like that

Comment (iliapolo) [reply examples/typescript/cdk8s-plus-elasticsearch-query/main.test.ts]: 👍

File: examples/typescript/cdk8s-plus-elasticsearch-query/data.txt (hunk 1 above)
Comment (iliapolo) [reply examples/typescript/cdk8s-plus-elasticsearch-query/data.txt]: 👍

**Pull request merged**